db/schema_tables.cc

```diff
@@ -2619,6 +2619,9 @@ static schema_mutations make_table_mutations(schema_ptr table, api::timestamp_ty
     if (table->is_counter()) {
         flags.emplace_back("counter");
     }
+    if (table->is_deletion_protection_enabled()) {
+        flags.emplace_back("deletion_protection_enabled");
```

Comment:
You're adding here a new feature to the schema tables, and we need to do it very carefully. Cassandra and older versions of Scylla might (?) have problems with it, we can have problems with partially-upgrading clusters exchanging schemas, and possibly with external tools and drivers which might not know what this means.

We have in Scylla a separate table for some Scylla-specific schema extensions, and also specifically in Alternator, we used in multiple places the idea of "tags" - these are special marks that you can add on tables that can say something and used to implement the DynamoDB concept of tags. We have several internal tags already like "system:ttl_attribute" which we use to store DynamoDB TTL settings, and "system:write_isolation" that we allow the user to set the write isolation mode of the table.

Can you please consider a table tag instead of inventing a whole new schema column? You can check out the existing uses of the above two tags in Alternator to see examples.

Reply:
Done. The table is protected by specifying a tag 'system:deletion_protection_attribute'; we can disable protection when the label is removed.